package.json

```diff
@@ -22,7 +22,7 @@
     "prepublish": "npm run build"
   },
   "devDependencies": {
-    "@playwright/test": "^1.38.1",
+    "@playwright/test": "^1.44.1",
```

Comment:
Note: Required a bump, because now we pass a ReadOnlyArray to toHaveText which we didn't support in 1.38.